src/components/ActivityView.vue

```diff
@@ -78,6 +78,11 @@ function getTransactionIcon(operation: TransactionOps | FileOps | TransakOps) {
     return getIconAsset('activities/tx-transfer-ownership.svg')
   }
 
+  const nearTransaction = ['Transaction', 'Batched Transaction']
```

Comment:
Classic set membership check, use `new Set(['Transaction', 'Batched Transaction'])` and `set.has`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#performance.

Reply:
Changes done here https://github.com/arcana-network/wallet-ui/pull/706/commits/24ab97ba39c3cb4b062b784899911786acd6b595